library/systemd/src/modules/systemd_service.rb

```diff
@@ -138,6 +136,20 @@ def all(propmap = {})
       end
     end
 
+    # Instantiate a SystemdService object based on the given name
+    #
+    # Use with caution as the service might exist or not. If you need to react when
+    # the service does not exist, use SystemdServiceClass.find.
+    #
+    # @param service_name [String] "foo" or "foo.service"
+    # @param propmap [SystemdUnit::PropMap]
+    # @return [Service] `nil` if not found
```

Comment:
I don't get the explanation of return value. Can it be nil?

Reply:
No, it cannot. Copy&paste error :)